.github/workflows/test.yaml

```diff
@@ -41,7 +41,7 @@ jobs:
         run: npm run build
 
       - name: Lint and test
-        run: npm test -- --runInBand
+        run: npm test -- --color --maxWorkers=2
```

Comment:
Adds colour to the CLI (as we did for `govuk-frontend`) but also uses 2x GitHub runner CPUs

Reply:
Is this some kind of bug/purposeful limit? I thought the Github Actions runners were all 2 cores+, so was this just not taking advantage of that for some reason?